unit-tests/test-phase.js

```diff
@@ -224,4 +223,19 @@ describe('phase', () => {
       assert.isFunction(phase.triggerGC);
     });
   });
+
+  describe('timeoutError', () => {
+    it('error thrown', () => {
+      let errorFunc = () => {
+        phase.timeoutError();
+      };
+      assert.throw(errorFunc, 'Test timeout exceeded ' + OPTIONS.timeout + 'ms');
```

Comment:
If we can get rid of the `let` declaration, then you can disregard the whitespace. Otherwise but a line between the variable declaration and the assert.